review-of-international-studies.csl

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="text" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Review of International Studies</title>
+    <id>http://www.zotero.org/styles/review-of-international-studies</id>
+    <link href="http://www.zotero.org/styles/review-of-international-studies" rel="self"/>
+    <link href="http://www.zotero.org/styles/urban-studies" rel="template"/>
+    <link href="https://www.cambridge.org/core/services/aop-file-manager/file/57753670dd5679630ae895f8" rel="documentation"/>
```

Comment:
I think it's a little easier if we point to https://www.cambridge.org/core/journals/review-of-international-studies/information/instructions-contributors instead for the "documentation" link. Likely to be a more stable link and it's easier to get to the journal's homepage from there.

Reply:
Otherwise no issues with style metadata or indentation.